src/pages/InstantaneousView/Formatters/InstantaneousFlowDataFormatter.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  InstantaneousFlowResponseData,
+  InstantaneousFlowDataSet,
+  InstantaneousFlowTableDescription,
+  InstantaneousFlowDataItem,
+} from '../../../types/api';
+import {
+  DataTableDataType,
+  DataTableHeader,
+  FormattedData,
+  InstantaneousFlowPageData,
+  InstantaneousFlowTableHeaderName,
+} from '../../../types/tables';
+import dayjs from 'dayjs';
+
+export const getTableHeaderName = (
+  tableName: InstantaneousFlowTableDescription
+): InstantaneousFlowTableHeaderName => {
+  switch (tableName) {
+    case InstantaneousFlowTableDescription.zoneSupply:
+      return InstantaneousFlowTableHeaderName.systemEntry;
+    case InstantaneousFlowTableDescription.terminalSupply:
```

Comment:
Should this have the same header as the one below?

Reply:
Apparently not... Currently isn't on their site anyway. Will probs change when we get designs